### PR TITLE
Add Smoothcomp

### PR DESCRIPTION
**Pull request by soodaayush**

### CODE REVIEW

File: entries/s/smoothcomp.com.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "Smoothcomp": {
+    "domain": "smoothcomp.com",
+    "contact": {
+      "twitter": "smoothcomps"
+    },
+    "categories": [
+        "tickets"
```
Comment (hkamran80) [inline entries/s/smoothcomp.com.json]:
```suggestion
      "tickets"
```

File: img/s/smoothcomp.com.svg [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	<path fill="#FFFFFF" d="M414.04,49.109h-6.376V18.974h8.675c6.854,0,11.32,2.517,11.32,9.353c0,6.395-3.773,9.512-10.453,9.512
+		h-3.166V49.109z M414.04,32.962h1.301c3.773,0,5.812-0.759,5.812-4.676c0-3.877-2.429-4.437-5.985-4.437h-1.128V32.962z"/>
+</g>
+<g>
+	<path fill="#FFFFFF" d="M49.434,66.167h-30.65c-9.47,0-17.175-7.705-17.175-17.176V18.342c0-9.471,7.705-17.175,17.175-17.175
+		h30.65c9.47,0,17.175,7.705,17.175,17.175v30.649C66.609,58.462,58.904,66.167,49.434,66.167z M18.784,6.988
+		c-6.261,0-11.354,5.093-11.354,11.354v30.649c0,6.262,5.093,11.354,11.354,11.354h30.65c6.261,0,11.354-5.093,11.354-11.354V18.342
+		c0-6.261-5.093-11.354-11.354-11.354H18.784z"/>
+</g>
+</svg>
```
Comment (hkamran80) [inline img/s/smoothcomp.com.svg]: Here's a minified SVG

```suggestion
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 89.3 89.3"><path d="M29.1 60.5h31.6V29H29zm34.4 8H26.3a5.2 5.2 0 0 1-5.2-5.1V26c0-2.8 2.3-5.1 5.2-5.1h37.2a5.1 5.1 0 0 1 5.2 5.1v37.3c0 2.8-2.3 5.1-5.2 5.1"/><path d="M23.6 8C15 8 8 15 8 23.6v42.1c0 8.6 7 15.6 15.6 15.6h42.1c8.6 0 15.6-7 15.6-15.6V23.6C81.3 15 74.3 8 65.7 8Zm42.1 81.3H23.6C10.6 89.3 0 78.7 0 65.7V23.6C0 10.6 10.6 0 23.6 0h42.1c13 0 23.6 10.6 23.6 23.6v42.1c0 13-10.6 23.6-23.6 23.6"/></svg>
```

**Review by hkamran80 — CHANGES_REQUESTED** [review]

**Comment by soodaayush** [comment]
Made the changes

Just a question: How do you get the minified version of an SVG?

**Comment by hkamran80** [comment]
> Just a question: How do you get the minified version of an SVG?

If I need to resize the SVG (e.g. remove text), I use [Inkscape](https://inkscape.org/). Then I run the SVG through [SVGOMG](https://jakearchibald.github.io/svgomg/) with all settings turned on, except for "Remove xmlns". I also adjust the precision sliders until they're as low as possible while retaining the original quality.

**Review by hkamran80 — APPROVED** [review]

**Pull request merged**